src/graphic/Fast3D/gfx_direct3d11.cpp

```diff
@@ -390,6 +390,10 @@ void CSMain(uint3 DTid : SV_DispatchThreadID) {
     SohImGui::Init(window_impl);
 }
 
+static int gfx_d3d11_get_max_texture_size() {
+    return 16384;
```

Comment:
Isn't this hardware dependent? Doesn't d3d have a way to query this? We should be doing that rather than hard coding.

Reply:
`D3D11_REQ_TEXTURE2D_U_OR_V_DIMENSION ` would be better than a hardcoded value.
https://learn.microsoft.com/en-us/windows/win32/direct3d11/overviews-direct3d-11-resources-limits

It looks like this might be the minimum that hardware needs to support to be D3D11 compatibility.